task/06-conditions-n-loops-tasks.js

```diff
@@ -431,27 +511,46 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+  const winPos = [
+    [[0, 0], [0, 1], [0, 2]],
+    [[1, 0], [1, 1], [1, 2]],
+    [[2, 0], [2, 1], [2, 2]],
+    [[0, 0], [1, 0], [2, 0]],
+    [[0, 1], [1, 1], [2, 1]],
+    [[0, 2], [1, 2], [2, 2]],
+    [[0, 0], [1, 1], [2, 2]],
+    [[0, 2], [1, 1], [2, 0]]
+  ];
+
+  for (let i of winPos) {
+    if (
+      position[i[0][0]][i[0][1]] &&
+      position[i[0][0]][i[0][1]] === position[i[1][0]][i[1][1]] &&
+      position[i[0][0]][i[0][1]] === position[i[2][0]][i[2][1]]
+    ) {
+      return position[i[0][0]][i[0][1]];
+    }
+  }
```

Comment:
It's hard to understand what's going on here. You have 2 paths:
1) Document what is `winPositions` and how this algorithm works
2) Create a few additional variables which names tells us what they stores and this names should help us to understand what algorithm exactly do.